packages/backend/src/open_payments/quote/service.ts

```diff
@@ -268,6 +283,70 @@ function calculateFixedSendQuoteAmounts(
   }
 }
 
+// WIP: rework to
+// debitAmount = 500
+// sourceAmount/debitAmountMinusFees = 500 - (500 * 0.02 + 100) = 390
+// receiveAmount = floor(debitAmountMinusFees * estimatedExchangeRate) = floor(390 * 0.91) = 354
```

Comment:
We wanted to rework `calculateFixedSendQuoteAmounts` to be based off the sending amount as noted here. I started to implement it but was getting some off by 1 errors. messed with how we ceiled/floored a bit with no difference but its probably worth trying again and accounting for the difference (should still work in theory, i think)